tests/config/cratedb.ini

```diff
@@ -1,14 +1,20 @@
 [Datasource]
 Name = cratedb
-LogicalQueryFormat = simple_lower
+LogicalQueryFormat = cratedb
 SchemaName = doc
 CommandLineOverride = FILL_ME_WITH_PATH_OF_REPOSITORY
+MaxThread = 6
```

Comment:
Maybe this setting should be documented. Could it be too high, depending on your environment?